packages/AutoTDD-GUI.package/ATDDOverview.class/class/messageDictionary.st

```diff
@@ -0,0 +1,10 @@
+instance creation
+messageDictionary
+	
+	^ IdentityDictionary
+		newFrom: {
+			#error -> 'Errors in'.
+			#failure -> 'Failures in'.
+			#pending -> 'unknown'.
+			#successful -> 'All passed'
```

Comment:
If we have capital letters on 3 of 4, we should probably use a capital letter on the 4th too:

```suggestion
			#error -> 'Errors in'.
			#failure -> 'Failures in'.
			#pending -> 'Unknown'.
			#successful -> 'All passed'
```